rdflib/plugins/stores/sparqlconnector.py

```diff
@@ -162,7 +163,8 @@ def update(
             raise SPARQLConnectorException("Query endpoint not set!")
 
         params = {}
-
+        
+        default_graph = default_graph or self.kwargs.get("default_graph")
```

Comment:
```suggestion
        default_graph = default_graph or self.kwargs.get("default_graph")
```

Reply:
Ruff is complaining about an extra blank line here, that is what the remaining failing test in the CI is.